raisinbread/RB_files/RB_Config.sql

```diff
@@ -19,7 +19,7 @@ INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (16,20,'false');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (17,21,'true');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (18,22,'true');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (19,23,'0');
-INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (20,24,'Modify this to your project\'s citation policy');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (20,24,"Modify this to your project\'s citation policy");
```

Comment:
Why is there a slash `project\'s`

Reply:
because phpstorm hates me

Reply:
undid the change

Reply:
there should be a `\` in `project\'s` because `'` is the delimiter that the SQL standard uses for strings.

Reply:
@ridz1208 then it should be single not double quotes.

Reply:
IT ISSSS

Reply:
@ridz1208 I see `"Modify this to your project\'s citation policy"` not `'Modify this to your project\'s citation policy'`

Reply:
refresh